lib/src/client/json_api_client.dart

```diff
@@ -66,7 +66,7 @@ class JsonApiClient {
   ///
   /// https://jsonapi.org/format/#crud-creating
   Future<Response<ResourceData>> createResourceAt(Uri uri, Resource resource,
-          {Map<String, String> headers}) =>
+          {Map<String, String> headers, Map<String, Object> meta}) =>
```

Comment:
Looks like the argument is not passed down to `_resourceDoc()`.
Please add a unit test supporting this change. It would belong in this folder https://github.com/f3ath/json-api-dart/tree/master/test/unit/client
You can use the ones that are there already as an example.

Reply:
OK, i will fixed it, thank you!

Reply:
should I use naming **documentMeta** to replace **meta**? 🤔

Reply:
No. Please disregard that comment. I removed it once realized that resource meta would be in the resource object itself. So just meta is OK.

Reply:
Get it!